zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
+    and paste the resulting authorization code.
+
+1.  Optionally, pass [commandline arguments](#configuration-options) to
+    configure the integration. The commandline arguments override the
+    corresponding settings in the `.zuliprc` file.
+
+    ```
+    python {{ integration_path }} \
+    --calendar primary \
+    --interval 3600 \
+    --client-secret-file /home/username/zulip/integrations/client-secret-credentials.json \
+    --tokens-file /home/username/zulip/integrations/google-oauth-tokens.json \
+    --channel core-team \
+    --topic scheduled-events \
+    --format-message "The event [{title}]({calendar_link}), at {location}, is from {start} to {end}.\n> {description}\n\n[Join call]({google_meet_link})."
+    ```
+
+    See [configuration options](#configuration-options) for the list of
+    available options.
+
+1.  You will get reminders as long as the terminal session with the bot
+    remains open. Consider using `screen` to run the bot in the background.
+
+!!! tip ""
+
+    Newly added calendar events may take up to 10 minutes to generate
+    notifications.
+
+{end_tabs}
 
-1.  Next, follow the instructions for **Step 1** at
-    [this link](https://developers.google.com/google-apps/calendar/quickstart/python)
-    to get a `client_secret` file. Save this file as `client_secret.json`
-    to your `~/` directory.
+### Configure the integration
 
-1.  Next, install the latest Google API Client for Python by following the
-    instructions on the
-    [Google website](https://developers.google.com/api-client-library/python/start/installation).
+The integration can be configured by:
 
-1.  In Zulip, go to your click on the cog in the top right corner, and
-    then clicking on **Personal settings**.
+* Passing command line arguments to the integration script.
+* Editing the `google-calendar` section of the `zuliprc` file.
 
-1.  Click on the tab that’s labeled **Account & privacy** and click on
-    **Manage your API key**. Enter your password if prompted, and
-    download the `zuliprc` file. Save this file as `.zuliprc` to your `~/`
-    directory.
+The configuration settings in `zuliprc` will be overridden by the
+corresponding command line options, if both are used.
 
-    ![Download zuliprc file](/static/images/integrations/google/calendar/001.png)
+### Configuration options
 
-1.  Run the `get-google-credentials` with this command:
+The integration script accepts the following configuration options:
 
-        python /usr/local/share/zulip/integrations/google/get-google-credentials
+* `calendar`: The `calendar ID` of the Google calendar to get events from.
+By default, the `primary` calendar is used.
 
-1.  It should open up a browser and ask you for certain permissions. Give
-    Zulip access, and move on to the next step. If it doesn’t open a
-    browser, follow the instructions in the terminal window.
+* `interval`: How many minutes in advance you want reminders delivered.
+The default value is 30 minutes.
```

Comment:
So, in the examples above, this is set to 3600, which I'm guessing is an hour? This doesn't make it clear that the interval needs to be in seconds and not minutes.